packages/amplify-e2e-tests/src/__tests__/function_9.test.ts

```diff
@@ -214,6 +214,7 @@ describe('nodejs', () => {
       await initJSProjectWithProfile(projRoot, {});
       await addApi(projRoot, {
         IAM: {},
+        transformerVersioN: 2,
```

Comment:
Looks like a typo snuck in: `transformerVersioN` (note the uppercase N at the end).

Reply:
thanks, noticed in a failed test, updated